### ISSUE TITLE
Consider adding an overall description

### ISSUE THREAD

**Issue by cordawso** [cgpriceisright/website#1]
I like the art and design, but the first words I'm seeing are about acoustic guitars. I'd like to know first just a couple sentences about what the site is first?

**Comment by lexinal** [comment]
I agree with Cory. I like the art and design as well. As I was clicking through your page, it was confusing on what the intent of the site is for. I assume it is about guitars, but what about these guitars? The pictures help, but it is important to connect the dots for users.
